src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1567,6 +1585,14 @@ public CollectionName ExtractCollectionName(DocumentsOperationContext context, s
             if (collections == null)
                 throw new InvalidOperationException("Should never happen!");
 
+            // If collection doesn't exist yet: Capitalize first letter *only if* all letters are lower case
+            if (collectionName.Any(char.IsUpper) == false)
```

Comment:
We don't want to force that on the server side.

Reply:
Moved to Studio